stories/Molecules/Blocks/DownloadCard/DownloadCard.scss

```diff
@@ -10,16 +10,64 @@
   }
 
   &:hover {
-    .publication_thumbnail__image {
+    .publication-thumbnail__image {
       &::after {
         opacity: 0.75;
       }
+  
+      &.yellow {
+        &::after {
+          background-image: linear-gradient(42deg, $color-yellow, transparent 80%);
+        }
+      }
+  
+      &.red {
+        &::after {
+          background-image: linear-gradient(42deg, $color-red, transparent 80%);
+        }
+      }
+  
+      &.green {
+        &::after {
+          background-image: linear-gradient(42deg, $color-green, transparent 80%);
+        }
+      }
+  
+      &.blue {
+        &::after {
+          background-image: linear-gradient(42deg, $color-azure, transparent 80%);
+        }
+      }
```

Comment:
@neelesh-tippe Write the above lines of code as:
```
$colors: ("yellow": $color-yellow, "red": $color-red, "green": $color-green, "blue": $color-azure);

@each $name, $color in $colors {
  &.#{$name}:after {
    @include background-image(42deg, #{$color}, transparent 80%);
  }
}
```
This will compile to:
```
&.yellow:after {
  background: linear-gradient(42deg, #ffeb00, transparent 80%);
}
&.red:after {
  background: linear-gradient(42deg, #ee402d, transparent 80%);
}
&.green:after {
  background: linear-gradient(42deg, #6de354, transparent 80%);
}
&.blue:after {
  background: linear-gradient(42deg, #60d4f2, transparent 80%);
}
```

Reply:
Not working for publication module. Need help.